uint16[] data			# [X, Y, orientation, size]uint16[]	data		# [X, Y, orientation, size]
